drivers/n25q128/n25q128.c

```diff
@@ -0,0 +1,293 @@
+/*
+ * Copyright (C) 2019 HAW Hamburg
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers
+ * @{
+ *
+ * @file
+ * @brief       Driver for the N25Q128 serial flash memory
+ *
+ * @author      René Herthel <rene-herthel@haw-hamburg.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+#include "n25q128.h"
+#include "include/n25q128_internal.h"
+
+/*
+ * @brief   Clamps a specifc value between minimum and maximum.
+ */
+#define CLAMP(x, min, max)    ((x > max) ? max : ((x < min) ? min : x))
+
+/*
+ * @brief   Calls the spi driver for receiving bytes.
+ */
+static inline void _receive(spi_t bus, uint8_t * in, size_t len)
+{
+    spi_transfer_bytes(bus, SPI_CS_UNDEF, false, NULL, in, len);
```

Comment:
Why do you need to wrap around the SPI call? And why `SPI_CS_UNDEF`? You don't need to handle CS yourself.